src/omero_metadata/populate.py

```diff
@@ -1252,6 +1379,39 @@ def post_process(self):
             elif (ScreenI is target_class or PlateI is target_class):
                 log.info('Missing plate name column, skipping.')
 
+            if roi_column is not None and (
+                    ImageI is target_class and
+                    resolve_roi_names and not resolve_roi_ids):
+                rname = ""
+                try:
+                    log.debug(roi_column)
+                    rid = roi_column.values[i]
+                    rname = self.value_resolver.get_roi_name_by_id(rid)
+                except KeyError:
+                    log.warn(
+                        "%d not found in roi ids" % rid)
+                assert i == len(roi_name_column.values)
+                roi_name_column.values.append(rname)
+                roi_name_column.size = max(
+                    roi_name_column.size, len(rname))
+            elif roi_name_column is not None and (
+                    ImageI is target_class and
+                    resolve_roi_ids and not resolve_roi_names):
+                if self.value_resolver.ambiguous_naming:
+                    raise MetadataError('Cannot resolve ROI names.')
+                rid = -1
+                try:
+                    log.debug(roi_name_column)
+                    rname = roi_name_column.values[i]
+                    rid = self.value_resolver.get_roi_id_by_name(rname)
+                except KeyError:
+                    log.warn(
+                        "%d not found in roi names" % rname)
+                assert i == len(roi_column.values)
+                roi_column.values.append(rid)
+            else:
+                log.info('No ROI information resolution needed, skipping.')
```

Comment:
```suggestion
                log.debug('No ROI information resolution needed, skipping.')
```

As INFO is the default debugging level, having this line printed every row if you have a CSV with ROI ID and names populated is a bit too verbose.